Sources/Mocker.swift

```diff
@@ -75,11 +80,20 @@ public struct Mocker {
     }
     
     /// Retrieve a Mock for the given request. Matches on `request.url` and `request.httpMethod`.
-    ///
+    /// if `onMockFor` is set, first see if that closure returns a mock; if it does, use that mock.
+	/// Otherwise, look for a specific mock that has been registered for this request.
+	/// If not found, check for a generic file extension mock.
+	///
     /// - Parameter request: The request to search for a mock.
     /// - Returns: A mock if found, `nil` if there's no mocked data registered for the given request.
     static func mock(for request: URLRequest) -> Mock? {
         shared.queue.sync {
+			/// Zeroth support for dynamic mocks via `onMockFor` closure
```

Comment:
Was the use of Zeroth on purpose or a typo?

Reply:
on purpose, but misguided 2am decision :) Fixed. Thanks.